ci/travis/build_project.sh

```diff
@@ -2,8 +2,4 @@
 
 set -e
```

Comment:
Since the script is reduced to one command, it can be removed, and call `make` directly here:
https://github.com/analogdevicesinc/libtinyiiod/blob/5f51d753fff55549958cb419f43b17e8ebde17dc/ci/travis/run_build.sh#L7

Reply:
Agree